src/v2/view-builder/internals/BaseView.js

```diff
@@ -20,7 +20,7 @@ export default View.extend({
     // Create Model
     const IonModel = this.createModelClass();
     const model = new IonModel ({
-      formName: this.options.remediationValue.name,
+      formName: this.options.currentViewState.name,
```

Comment:
Maybe we should just call this `this.options.currentViewState.formName` to be clear?

Reply:
ah, nvm i think this is what the api sends us right ? formName comes under the `name` property

Reply:
`currentViewState` is from `remediation` hence it is `name'`